ocs_ci/ocs/resources/catalog_source.py

```diff
@@ -18,7 +18,8 @@ class CatalogSource(OCP):
     methods we need to do with it.
     """
 
-    def __init__(self, resource_name="", *args, **kwargs):
+    def __init__(self, resource_name="", resource_namespace="", *args,
```

Comment:
Also please keep just namespace parameter as it's the same which OCP object is using and this class inherit from it. 

We are directly passing namespace also here: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/deployment.py#L226, so if you would like to explicitly define, use `namespace` name of parameter.

Reply:
Done